modules/module-info.php

```diff
@@ -625,3 +625,18 @@ function jetpack_xml_sitemap_more_info() {
 }
 add_action( 'jetpack_module_more_info_sitemaps', 'jetpack_xml_sitemap_more_info' );
 // XML Sitemap: STOP
+
+/**
+ * Google Analytics
+ */
+function jetpack_google_analytics_more_link() {
+	echo 'https://support.wordpress.com/google-analytics/';
```

Comment:
Brandon, there's not a page at https://jetpack.com/support/google-analytics yet.